tests/_helpers.py

```diff
@@ -117,8 +117,8 @@ def any_string(expected):
 
 
 def any_timestamp(expected):
```

Comment:
`any_int`

there's nothing special about timestamp here, other than intended semantics

Reply:
Good point. Added a c heck to verify the order of value magnitude.

Reply:
This heuristic helps a bit but it's not super robust.

Could we maybe get some inspiration from https://stackoverflow.com/questions/12422918/how-to-validate-timestamp-in-javascript?